### PR TITLE
Hamburger

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Make HomePage Responsive.

Added Hamburger for screens less than or equal to 1000px

### CODE REVIEW

File: src/Screens/HomeScreen/HomeScreen.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { useEffect } from "react";
 import { getPosts } from "../../actions/postsAction";
+import Hamburger from "../../Components/Hamburger/Hamburger";
 import PostContent from "../../Components/PostContent/PostContent";
 import ProfileCard from "../../Components/ProfileCard/ProfileCard";
 import SideBar from "../../Components/SideBar/SideBar";
```
Comment (sushil-mod) [inline src/Screens/HomeScreen/HomeScreen.jsx:6]: you  can try barrel exports (by creating one index file inside components folder )

File: src/Components/Icon.js [hunk 2]
```diff
@@ -1,6 +1,18 @@
 import { AiOutlineHome, AiOutlineLike } from "react-icons/ai";
```
Comment (sushil-mod) [inline src/Components/Icon.js:1]: i liked how created seprated file for only icons

**Review by sushil-mod — APPROVED** [review]
Some suggestion . Hamburger functionality working fine and responsiveness is also good . good work .

File: src/Screens/HomeScreen/HomeScreen.jsx (hunk 1 above)
Comment (munjalSaksham71) [reply src/Screens/HomeScreen/HomeScreen.jsx:6]: will do it in next PR.

File: src/Components/Hamburger/Hamburger.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import { useAuth } from "../../context/auth-context";
+
+const Hamburger = () => {
+  const { logoutUser } = useAuth();
+
+  return (
+    <div className="hamburger flex-column pt-3">
+      <Link to="/" className="list_item flex-row p-3">
+        Home
+      </Link>
```
Comment (t007rushi) [inline src/Components/Hamburger/Hamburger.jsx]: 1.this is a repeating line of code you can map an array with id, link,title as if you will increase options in hamburger in future 

2.also benefits you can also add Icon in menu

**Review by t007rushi — APPROVED** [review]
-> some minor suggestions in UI:
align Navbar items like title and ham icon

 -> Increase the font size of Text
-> add link to Logo Airpark

File: src/Components/Hamburger/Hamburger.jsx (hunk 3 above)
Comment (munjalSaksham71) [reply src/Components/Hamburger/Hamburger.jsx]: Oh Yeah, I forgot . thanks For suggesting.. Will do it now.

**Pull request merged**